_posts/wiki/2018-10-18-ROS-kinetic-installation.md

```diff
@@ -73,15 +73,16 @@ Now, there are a number of options to choose from. The recommended one is the _D
 
 ~~~bash
 rosinstall_generator desktop_full --rosdistro kinetic --deps --wet-only --tar > kinetic-desktop-full-wet.rosinstall
-wstool init -j8 src kinetic-desktop-full-wet.ros
+wstool init -j8 src kinetic-desktop-full-wet.rosinstall
```

Comment:
Isn't this already fixed? Because I see it fixed if I look at here https://hiro-group.github.io/ros_kinetic_installation.html